integrations/datadog/.port/resources/port-app-config.yaml

```
@@ -0,0 +1,73 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: host
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"datadogHost"'
+          identifier: .id | tostring
+          title: .name
+          properties:
+            up: .up
+            hostName: .host_name
+            description: .description
+            metrics: .metrics
+            sources: .sources
+  - kind: monitor
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"datadogMonitor"'
+          identifier: .id | tostring
+          title: .name
+          properties:
+            tags: .tags
+            monitorType: .type
+            overallState: .overall_state
+            thresholds: .thresholds
+            priority: .priority
+            createdBy: .creator.name
+            createdAt: .created
+            updatedAt: .modified
+  - kind: serviceCatalog
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"datadogServiceCatalog"'
+          identifier: .attributes.schema."dd-service"
+          title: .attributes.schema."dd-service"
+          properties:
+            contacts: .attributes.schema.contacts
+            languages: .attributes.schema.languages
+            description: .attributes.schema.description
+            tags: .attributes.schema.tags
+            type: .attributes.schema.type
+            code_repositories: .attributes.schema.links | map(.url)
+  - kind: slo
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          blueprint: '"datadogSlo"'
+          identifier: .id | tostring
+          title: .name
+          properties:
+            tags: .tags
+            sloType: .type
+            description: .description
+            warningThreshold: .warning_threshold
+            targetThreshold: .target_threshold
+            createdBy: .creator.name
```

```suggestion
            createdBy: .creator.email
```

we thrive to manage the user information as an email to make it easy to connect it later the its user in port

---

Alright, i have corrected this